eth/beacon/types/validator_records.py

```diff
@@ -52,3 +55,13 @@ def __init__(self,
             latest_status_change_slot=latest_status_change_slot,
             exit_count=exit_count,
         )
+
+    @property
+    def is_active(self) -> bool:
+        """
+        Returns ``True`` if the validator is active.
+        """
+        return self.status in [
```

Comment:
The `[ACTIVE, PENDING_EXIT]` structure should 1) be a `set` for faster inclusion checks and 2) be elevated to a top level module constant to avoid runtime object creation overhead.

Reply:
Resolved. Thanks.